powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/SignatureMetadata.java

```diff
@@ -0,0 +1,15 @@
+package io.getlime.security.powerauth.app.server.database.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class SignatureMetadata {
```

Comment:
I think this should be an implementation of the interface - we need to use different types eventually.

Reply:
The specific class might need to have also specific JSON serialization / deserialization annotation, plus i.e., a discriminator transient property.